Dockerfile.deploy

```diff
@@ -0,0 +1,54 @@
+# docker build --build-arg VNC_PASSWORD=<secret> -f Dockerfile.deploy -t registry.esss.lu.se/ecdc/ess-dmsc/daqlite:<x.x> .
+
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV USER=root
+ENV DISPLAY=:1
+ARG VNC_PASSWORD=password
+ENV VNC_PASSWORD=${VNC_PASSWORD}
+ENV LD_LIBRARY_PATH="/ess/ecdc/daqlite/lib"
+
+# Install necessary packages
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        tightvncserver \
+        xfce4 \
+        xfce4-goodies \
+        qtcreator \
+        qtbase5-dev \
+        qt5-qmake \
+        vim-tiny \
+        x11-xserver-utils \
+        gedit \
+        xfonts-base \
+        xfonts-75dpi \
+        dbus-x11 && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+
+# Create directories
+RUN mkdir -p /ess/ecdc/daqlite/build /ess/ecdc/daqlite/configs
+
```

Comment:
I would create this dummy kafka config to prevent erros if daqlite looing for it.

```suggestion
RUN echo "{}" > /ess/ecdc/daqlite/configs/kafka-config-daqlite.json

```